### ISSUE TITLE
[Bug]: v0.6.3 "[!] unimplemented code!!!"

### ISSUE THREAD

**Issue by JesseWV** [LavaGang/MelonLoader#678]
### All of the following criteria must be met

- [X] All <a href="https://github.com/LavaGang/MelonLoader/blob/master/README.md#requirements">Requirements</a> must be installed.
- [X] Changed the title so that it doesn't just says "[Bug]: "
- [X] I have searched the GitHub issues for my bug, even in the closed issues.

### All of the following are optional to answer

- [ ] Tried reinstalling the Game.
- [X] Tried reinstalling MelonLoader.
- [X] Tried restarting PC.
- [ ] Was able to see the Start Screen.

### Describe the issue.

Using MelonLoader v0.6.3 and game client 2.28 136607 S, I get the error below followed by immediate exit. The game loads and does not produce the same error upon reverting to v0.6.2.  Logs and screenshot also below.

```
[!] [C:\Users\Sam\CLionProjects\Dobby\source\InstructionRelocation\x64\X64InstructionRelocation.cc:86:GenRelocateCodeFixed]: 
[!] unimplemented code!!!
```

The log files do not show this error:

Latest.log
```
[16:11:38.827] [DEBUG] Initializing Analytics Blocker...
[16:11:38.847] [DEBUG] Hooking wsock32::gethostbyname (0x7FFC5BF094C0)...
[16:11:38.987] [DEBUG] Hooking ws2_32...
```

Latest-Bootstrap.log
```
[16:11:38.674] [DEBUG] Attaching hook to il2cpp_init
[16:11:38.747] [DEBUG] Detaching hook from il2cpp_init
[16:11:38.791] [DEBUG] [Dotnet] Invoking LoadStage1
[16:11:38.795] [DEBUG] [Dotnet] Reloading NativeHost into correct load context and getting LoadStage2 pointer
[16:11:38.797] [DEBUG] [Dotnet] Invoking LoadStage2

```
![MelonLoader 0 6 3 Error](https://github.com/LavaGang/MelonLoader/assets/6340641/d86c17c1-196a-419e-a9f9-e64d4538a4a1)

### Did you attach your log file?

Yes, I attached my log file to the text box above.

**Comment by KibblesTheKitten** [comment]
Not a single mod works right now. Please be patient. And the unimplemented code issue will soon be fixed in ml 0.6.3

**Comment by JesseWV** [comment]
> Not a single mod works right now. Please be patient. And the unimplemented code issue will soon be fixed in ml 0.6.3

Understood and of course that is to be expected. That being said, I'm not sure this is isolated to the new DLC release as I saw a reports on Reddit of this happening yesterday before it's release: 
https://www.reddit.com/r/thelongdark/comments/1dn8nk5/issue_and_error_with_first_install_of_melon/

**Comment by HerpDerpinstine** [comment]
> > Not a single mod works right now. Please be patient. And the unimplemented code issue will soon be fixed in ml 0.6.3
> 
> Understood and of course that is to be expected. That being said, I'm not sure this is isolated to the new DLC release as I saw a reports on Reddit of this happening yesterday before it's release: https://www.reddit.com/r/thelongdark/comments/1dn8nk5/issue_and_error_with_first_install_of_melon/


This is due to an issue with MelonLoader that was recently discovered
We will be pushing an update soon to address the issue

**Comment by HerpDerpinstine** [comment]
Fixed in v0.6.4
